docs/ADR-61-merkle-proofed-entities.md

```diff
@@ -0,0 +1,237 @@
+# Merkle proofed entities
+
+## Context and Problem Statement
+
+The Third Party Wearables initiative had the requirement of being able to publish collections containing a big amount of wearables.
+These big amount wearables would require multiple transactions to be sent and approved, incurring in a high cost in gas and in difficult UX experience.
+
+Taking in consideration this requirement, a new solution came up, described in the ADR-58 in where we will be able to publish collections containing a huge amount wearables with
+a single transaction using a Merkle Tree that will serve as proof for the wearable's metadata and contents to be uploaded into the Catalysts.
+
+These ADR describes a broader solution where all entities deployed to the Catalysts could be verified using a Merkle Tree.
+
+## Proposed solution
+
+In order to avoid changing the entity structure, the proposed solution will leverage the entities' metadata by including a new property, `merkleProof`, that will contain everything
+needed to verify the entity against a Merkle Tree.
+
+The `merkleProof` property will be an object with the following schema:
+
+```json
+{
```

Comment:
could this be written as typescrypt type instead? for legibility

Reply:
I thought it was clearer in JSON, changed it to TS.